appium/tests/data/index.ts

```diff
@@ -31,6 +31,10 @@ export const CommonData = {
     subject: 'Test 1',
     message: 'Test email',
   },
+  keyMismatchEmail: {
+    subject: 'Encrypted message with key mismatch',
+    message: 'MIME-Version: 1.0',
```

Comment:
@tomholub
Unfortunately it is not possible to use full message text because selector does not work for such long text. Maybe we need to go away from selectors to some kind of ID.